### DISCUSSION TITLE
Remove Image Magick

### DISCUSSION THREAD

**Discussion by nicopicchio** [ImageMagick/ImageMagick#4642] [Help]
I have randomly found Image Magick installed on my PC. I have never heard of it and I am the only one using this PC.

Currently running Ubuntu 20.04 LTS.

Attempted to remove it via CLI with the following command:
`sudo apt remove imagemagick`
`sudo apt remove --purge imagemagick`

None of the above commands worked.

Can anybody advise on how to uninstall it please?

![Screenshot-20211230215435-2086x1135](https://user-images.githubusercontent.com/70696511/147790521-738c1392-3210-4c74-9277-18bca42ca299.png)

**Comment by fmw42** [comment]
It comes with Linux.  See make uninstall at https://imagemagick.org/script/advanced-linux-installation.php#build

**Reply by nicopicchio** [reply]
Thank you for your reply @fmw42 

I have tried to use `make uninstall` but this is what I got back:

```
nico@ubuntu-desktop:~$ make uninstall
make: *** No rule to make target 'uninstall'. Stop.
```

Any other option?

**Reply by fmw42** [reply]
You have to do ./configure then probably sudo make uninstall. But I am not really an expert on Linux install/uninstall. So one of the developers or experts will need to help you further.

**Comment by snibgo** [comment]
Does any software on your computer use ImageMagick? If it does, then removing IM will cause problems.

How do you normally install or remove software? That method may work for you. For example, I use Cygwin, and its package manager knows what software depends on what other software, so if I tell it to remove something that other software needs, it tells me.

Otherwise, you might simply delete the IM files. They are probably all in a single directory (or directory tree), and probably nothing else is in there.

Obviously, take backups before deleting stuff.

**Reply by nicopicchio** [reply]
Thank you @snibgo.
I am not sure if any of the computer software I have uses IM. To be honest I have only recently noticed the IM icon, which I am sure it was not there when I installed Ubuntu.
Could that be because I have installed something that requires IM and that software installed IM too?

**Comment by snibgo** [comment]
It is possible that you installed some software, and that software then installed IM.

**Reply by nicopicchio** [reply]
thank you @snibgo. that would make sense.

**Comment by jhcloos** [comment]
Running:

apt-cache rdepends --installed imagemagick

should show you what pulled it in as a dependency.

-JimC
-- 
James Cloos ***@***.***>         OpenPGP: 0x997A9F17ED7DAEA6

**Reply by nicopicchio** [reply]
Thank you @jhcloos, it is inkscape, which makes sense!

**Comment by pushpendra-407** [comment]
Hey @nicopicchio, you can refer to this link: https://askubuntu.com/questions/764553/how-to-uninstall-image-magick

I recently found a solution for removing ImageMagic packages.
